src/sentry/api/event_search.py

```diff
@@ -906,6 +906,76 @@ def validate_aggregate(field, match):
         )
 
 
+FUNCTION_PATTERN = re.compile(r"^(?P<function>[^\(]+)\((?P<columns>[^\)]*)\)$")
+
+FUNCTIONS = {
+    "percentile": {
+        "name": "percentile",
+        "args": [
+            {"name": "column", "type": "numeric_column"},
```

Comment:
In the front-end we're currently using 'number' and 'duration' as separate types to let the interface decide how to format number vs milliseconds and seconds. How would that work with this?

Reply:
These types are not well defined on the backend yet. I imagine ultimately they'll match the frontend and even possibly extend it. This is basically a placeholder that we can expand as we add more functions.

Reply:
Sounds good.